docs/installation.rst

```diff
@@ -77,45 +77,7 @@ Installation
 5. Migrate/Sync the database.
```

Comment:
With no 1.6 this could be "Migrate the database" and we could keep the "$ ./manage.py migrate name" line.